tools/coverage_plugins_package/setup.py

```diff
@@ -0,0 +1,26 @@
+import setuptools
+
+with open("README.md", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="coverage-plugins",
+    version="0.0.1",
+    author="Jane Xu",
+    author_email="janeyx@fb.com",
```

Comment:
make it same as top level setup.py?
```suggestion
    author='PyTorch Team',
    author_email='packages@pytorch.org',
```